static/js/components/subject_page/page_selector.tsx

```diff
@@ -23,14 +23,14 @@ import _ from "lodash";
 import React, { useEffect, useState } from "react";
```

Comment:
note: this should stay specific to topic pages.

Reply:
done